content/docker-hub/access-tokens.md

```diff
@@ -65,10 +65,16 @@ a password.
 You can rename, activate, deactivate, or delete a token as needed.
 
 1. Access your tokens under **Account Settings > Security**.
-   This page shows an overview of all your tokens. You can also view the number
+   This page shows an overview of all your tokens, and lists if the token was generated manually or if it was [auto-generated](#auto-generated-tokens). You can also view the number
    of tokens that are activated and deactivated in the toolbar.
 
-2. Choose a token and then select **Delete** or **Edit**, or use the menu on the far right of a token row to bring up the edit screen. 
+2. Select a token and then select **Delete** or **Edit**, or use the actions menu on the far right of a token row to bring up the edit screen.
    You can also select multiple tokens to delete at once.
 
-3. After modifying the token, select **Save**.
\ No newline at end of file
+3. After modifying the token, select **Save**.
+
+## Auto-generated tokens
+
+When you sign in to Docker Desktop, this generates an access token on your behalf. When you complete registry actions using the Docker CLI, the CLI uses this token for authentication. The token scope has Read, Write, and Delete access. If your Docker Desktop session expires, the token is automatically removed locally.
```

Comment:
```suggestion
When you sign in to your Docker account, Docker Desktop generates an authentication token on your behalf. When you interact with Docker Hub using the Docker CLI, the CLI uses this token for authentication. The token scope has Read, Write, and Delete access. If your Docker Desktop session expires, the token is automatically removed locally.
```

Reply:
Thanks @dvdksn - made those changes based on your suggestion, just modified it slightly to make it clear that you have to sign in to your account with DD for this token to generate.